### DISCUSSION TITLE
Please stop using the old logo / icon

### DISCUSSION THREAD

**Discussion by d-packs** [searx/searx#2785] [Ideas]
![Searx_logo svg](https://user-images.githubusercontent.com/33980111/116268260-d3277700-a785-11eb-9acb-b0088ce31157.png)

This logo is offensive for christian users. 

Not only that, but what's the message of the icon? "Evil searching"? That niche is already filled by google. It doesn't make sense message-wise and it hurts the audience levels. Why alienate a whole segment of the potential user base because of poor design decisions. 

There are alternative, modern variants of the searx logo, not involving demonic symbolism. Please use those and completely give this one up.

**Comment by cyclaero** [comment]
I see the Lynx. In many cultures a symbol for a prudently clever wild cat. A devils head got horns.
![266px-Lynx_lynx_poing](https://user-images.githubusercontent.com/13712142/116475977-efdcb100-a850-11eb-9c16-c472f5cfade6.jpg)

**Comment by silverwings15** [comment]
not religious myself but between the horn looking things and the one eye, i had the same reaction as OP when i first started using searx. i'm not particularly invested enough to ask for a change, but i certainly won't be opposed to one

**Comment by virtadpt** [comment]
I don't particularly care.  I use the API and the RSS feeds and not the web user interface.  If I need to, I'll set my own icon.

**Comment by keylogthis** [comment]
Well, I'm offended by Christians in general, so I'd be happy if they avoided it.

**Comment by emacsomancer** [comment]
Please bring back the old icon. It's so much more distinctive than the generic magnifying glass; similar icons are used elsewhere, so it's not very useful in being able to quick spot a Searx tab amongst others.
